ansible/roles/common/tasks/tools.yml

```diff
@@ -1,10 +1,19 @@
 ---
-- name: install wxtoimg
+- name: install wxtoimg arm
   become: yes
   apt:
     deb: "{{ noaa_home }}/software/wxtoimg-armhf-2.11.2-beta.deb"
     state: present
     update_cache: no
+  when: ansible_architecture == 'armv7l'
```

Comment:
huh, so we didn't support a 64bit OS for the pi 4 then?

Reply:
I've got the value from one of the pi I have that is probably an old v3 something, but I think armhf should run on a 64bit pi 4

Reply:
Apparently not? https://github.com/jekhokie/raspberry-noaa-v2/issues/463

Reply:
https://unix.stackexchange.com/questions/625576/how-to-run-32-bit-armhf-binaries-on-64-bit-arm64-debian-os-on-raspberry-pi I think it could be possible (unless/until the CPU drops entirely 32bit), but I don't have an arm64 device with such userland to test

but if it's not possible at all then wxtoimg fate is uncertain on arm64 :(